ingestion-edge/tests/test_api.py

```diff
@@ -11,14 +12,85 @@
 import pytest
 
 
-def test_heartbeat(client: SanicTestClient):
-    _, r = client.get("/__heartbeat__")
-    r.raise_for_status()
-    assert r.json == {
-        "checks": {"check_disk_bytes_free": "ok"},
-        "details": {},
-        "status": "ok",
-    }
+@dataclass
+class FakeStat:
+    f_bfree: int = 0
+    f_frsize: int = 0
+
+
+def test_heartbeat_noop(client: SanicTestClient):
+    def check(app):
+        assert app is client.app
+        return []
+
+    with patch("ingestion_edge.dockerflow.check_disk_bytes_free", new=check):
+        _, r = client.get("/__heartbeat__")
+        assert r.json == {
```

Comment:
I generally try to leave a context manager as soon as possible, similar to keeping the body of a `try` as short as possible. So, I would have this assert indented left here and in the other tests.

I don't evidence that this is a standard followed elsewhere, so just something to think about.

Reply:
i don't see how it would hurt, and i know reducing the size of try blocks is recommended